3-frames-and-windows/06-clickjacking/clickjacking-visible.view/facebook.html

```diff
@@ -3,7 +3,7 @@
 
 <body style="margin:10px;padding:10px">
 
-  <input type="button" onclick="alert('Like pressed on facebook.html!')" value="I LIKE IT !">
+  <input type="button" onclick="alert('facebook.html'de beğen butonuna basıldı!')" value="BEGENDIM">
```

Comment:
`BEGENDIM` -> `BEĞENDIM`

Reply:
Ğ harfinin kod içinde sorun yaratbileceğini düşünüyorum.

Reply:
Ne gibi bir sorun?

Reply:
Az önce string içinde olduğunu fark ettim. Benim hatam, sorun yok